provider/dummy/environs.go

```diff
@@ -981,7 +981,7 @@ func (e *environ) StartInstance(args environs.StartInstanceParams) (*environs.St
 	series := args.Tools.OneSeries()
 
 	idString := fmt.Sprintf("%s-%d", e.name, estate.maxId)
-	addrs := network.NewAddresses(idString+".dns", "127.0.0.1")
+	addrs := network.NewAddresses(idString+".dns", "127.0.0.1", "::1")
```

Comment:
I think your comment should go in the source code to explain why we don't use "localhost".